.github/workflows/get-changed-charts-merge-queue.yaml

```
@@ -0,0 +1,43 @@
+
+jobs:
+  getChangedCharts:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    outputs:
+      charts: ${{ steps.getCharts.outputs.charts }}
+      count: ${{ steps.getCharts.outputs.count }}
+    steps:
+      - name: Get all charts
+        if: ${{ inputs.pr_number }}
```

actionlint (1.7.4)

property "pr_number" is not defined in object type {base_commit: string; head_commit: string}

(expression)